packages/@aws-cdk/aws-eks/README.md

```diff
@@ -336,6 +336,33 @@ new KubernetesResource(this, 'hello-kub', {
 cluster.addResource('hello-kub', service, deployment);
 ```
 
+The following examples will deploy the [AWS ALB Ingress Controller](https://github.com/kubernetes-sigs/aws-alb-ingress-controller) 
```

Comment:
I am not sure the fully fledged example is what we need here. I would just go with something schematic like:

```ts
import * as yaml from 'js-yaml';
import * as request from 'sync-request';

const manifestUrl = 'https://url/of/manifest.yaml';
const manifest = yaml.safeLoadAll(request('GET', manifestUrl).getBody());
cluster.addResource('my-resource', ...manifest);
```

I think we should add first-class support for the AWS ALB Ingress Controller. Care to raise a GitHub issue with your code snippet?